static/app/views/issueList/searchBar.tsx

```diff
@@ -84,6 +93,35 @@ const getFilterKeySections = (
   ];
 };
 
+const getAdditionalIssueTags = (
```

Comment:
Add additional tags to the ones retrieved from the organization/org-slug/tags endpoint. These include the semver fields and the predefined builtIn tags.